### DISCUSSION TITLE
import collection data => update current collections

### DISCUSSION THREAD

**Discussion by jm61** [pocketbase/pocketbase#659] [Q&A]
Hello to all, 
with @faker-js/faker I can get big amount of datas for testing.
I'd include this data.json in my actual collections but it looks like it's not possible to do a partial import with "import collections" without deleting current collections... ?

**Comment by ganigeorgiev** [comment]
If you mean the "Sync" settings (aka. "Export Collections" and "Import Collections") - they are not for import/export of "records" (data). They are only for import/export of "collections", aka. your data structure, like fields, api rules, etc.

Import/Export of records currently is not supported from the UI, but it will be handled after the ongoing users refactoring and the support for bulk API transactions (you could watch issue #48).

You can insert your generated data programmatically by iterating over the json array and call `client.records.create("your_collection", { ... })` manually for each entry.

**Reply by jm61** [reply]
Your explanations are always crystal clear, direct to the point, thanks master Gani !
